soh/src/code/z_message_PAL.c

```diff
@@ -1281,8 +1281,15 @@ void Message_Decode(GlobalContext* globalCtx) {
     MessageContext* msgCtx = &globalCtx->msgCtx;
     Font* font = &globalCtx->msgCtx.font;
 
-    for (u32 i = 0; i < FONT_CHAR_TEX_SIZE * 120; i += FONT_CHAR_TEX_SIZE) {
-        gSPInvalidateTexCache(globalCtx->state.gfxCtx->polyOpa.p++, &font->charTexBuf[i]);
+    if (msgCtx->msgMode >= MSGMODE_OCARINA_STARTING && msgCtx->msgMode <= MSGMODE_OCARINA_AWAIT_INPUT || msgCtx->textBoxType == TEXTBOX_TYPE_OCARINA) {
```

Comment:
Not needed from a logic stand point, but it might be good for readability to use parens so it's more obviously `(a && b) || c` instead of `a && (b || c)`

Reply:
Added